uiautomator/__init__.py

```diff
@@ -474,10 +474,13 @@ def start(self, timeout=5):
         self.uiautomator_process = self.adb.cmd(*cmd)
         self.adb.forward(self.local_port, self.device_port)
 
-        while not self.alive and timeout > 0:
-            time.sleep(0.1)
-            timeout -= 0.1
-        if not self.alive:
+        step_secs = 0.1
```

Comment:
What are your thoughts about making this a defaulted argument?

A doc string would be good on this method too.